lib/exercises.rb

```diff
@@ -1,19 +1,55 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 The time/space complexity is correct if the strings are of limited size and O(n * m log m) if they are not (you're sorting the strings).